services/github/test/handler_test.js

```diff
@@ -738,6 +765,48 @@ helper.secrets.mockSuite(testing.suiteName(), ['azure'], function(mock, skipping
       assert(monitorManager.messages.some(({Type, Severity}) => Type === 'monitor.error' && Severity === LEVELS.err));
       monitorManager.reset();
     });
+
+    test('successfully adds custom check run text from an artifact', async function () {
```

Comment:
let's also have a test that tasks without custom check run stuff don't throw an error. it seems like we've made all tasks have this metadata in these tests now.

Reply:
> it seems like we've made all tasks have this metadata in these tests now.

Can you show me in the code where you see that?

Reply:
...because to me it seems what you're asking for is covered by the pre-existing test-cases, so I'm not sure what to do. It seems to me I am using a separate taskid for the tests for this functionality, but I could be wrong

Reply:
Just to confirm: I am implementing Dustin's suggestion on default name, and `Checks API: result status handler` _definitely_ doesn't have that metadata.

Reply:
ah, I see the switch-case handling what I am talking about. was that there before? if so I apologize for missing it